README.md

```diff
@@ -45,6 +45,11 @@ Change your directory back to `mattermost-plugin-apps` and run the end to end te
 make test-e2e
 ```
 
+## Usage
+Use `apps provision-aws url` to provision an app in AWS. Pass the url of the archived app. 
```

Comment:
Let's remove that for now as https://github.com/mattermost/mattermost-plugin-apps/pull/67 will introduce a seperate tool